tests/phpunit/compat/class-wc-stripe-subscription-renewal-test.php

```
@@ -485,6 +485,97 @@ public function test_renewal_radar_blocked_cancels_pending_retry() {
+		// Assert: the hook fired exactly once with the expected arguments.
+		$this->assertCount( 1, $hook_args );
+		$this->assertSame( $renewal_order->get_id(), $hook_args[0][0]->get_id() );
+		$this->assertNotEmpty( $hook_args[0][1]->error );
+		$this->assertSame( 'highest_risk_level', $hook_args[0][2] );
+
+		// Clean up.
+		remove_action( 'wc_stripe_subscription_renewal_blocked_by_radar', $listener, 10 );
+		remove_filter( 'pre_http_request', $pre_http_request_callback );
+		WC_Subscriptions_Helpers::$wcs_get_subscriptions_for_renewal_order = null;
+		WCS_Retry_Manager::mock_reset();
+	}
```

PHPMD (2.15.0)

[error] The method test_renewal_radar_blocked_fires_action_hook is not named in camelCase. (undefined)

(CamelCaseMethodName)